alpha/hal.cpp

```diff
@@ -247,6 +247,36 @@ void _TS_HAL::reset()
   ESP.restart();
 }
 
+void _TS_HAL::power_setPowerMode(TS_PowerMode powerMode) {
```

Comment:
have inidividual functions for each feature; what we decide to power-down depends on higher level state machine.

Reply:
also dont mix camel case with snake case.
all function names here are snake cased.